lightweight_charts/abstract.py

```diff
@@ -314,7 +314,7 @@ def __init__(self, volume_enabled: bool = True, inner_width: float = 1.0, inner_
         self._position = 'left'
         self.loaded = False
         self._html = HTML
-        self._scripts = []
```

Comment:
Had to rename this as it coincides with `._scripts` on the `ReactiveHTML` class.